server/controllers/visitation.controllers.js

```diff
@@ -3,32 +3,38 @@ import EMAIL_CONTENT from '../../mailer';
 import { sendMail } from '../services/mailer.service';
 import httpStatus from '../helpers/httpStatus';
 
-const BALLERS_EMAIL = process.env.BALLERS_EMAIL || 'dev@highrachy.com';
-
 const VisitationController = {
   book(req, res, next) {
     const booking = req.locals;
     const { user } = req;
     scheduleVisitation({ ...booking, userId: user._id })
-      .then((schedule) => {
+      .then(({ schedule, vendorEmail }) => {
         const contentBottom = `
           <strong> Name: </strong>: ${schedule.visitorName}<br /> 
           <strong> Phone: </strong> ${schedule.visitorPhone}<br />
           <strong> Email: </strong> ${
             schedule.visitorEmail ? schedule.visitorEmail : 'No email provided'
           } <br />
         `;
-        sendMail(EMAIL_CONTENT.SCHEDULE_VISIT, { email: BALLERS_EMAIL }, { contentBottom });
+        sendMail(EMAIL_CONTENT.SCHEDULE_VISIT, { email: vendorEmail }, { contentBottom });
         res
           .status(httpStatus.CREATED)
           .json({ success: true, message: 'Visit scheduled successfully', schedule });
       })
       .catch((error) => next(error));
   },
+
   getAll(req, res, next) {
-    getAllVisitations()
+    const userId = req.user._id;
```

Comment:
We can send the whole user object instead of getting the `_id` only here.
```suggestion
    const { user } = req;
```